### PR TITLE
Implement changes for API 1.5

### PR DESCRIPTION
**Pull request by kirkplangrid**

### CODE REVIEW

**Comment by kirkplangrid** [comment]
@josephdwyer if you could also look at this commit I accidentally checked into dev directly:  https://github.com/plangrid/plangrid-api-net/commit/bbcf831833fccc1e82e90d8d4c81872f0b4a864c

File: PlanGrid.Api/RfiStatus.cs [hunk 1]
```diff
@@ -1,4 +1,8 @@
-﻿using Newtonsoft.Json;
+﻿// <copyright file="RfisTests.cs" company="PlanGrid, Inc.">
+//     Copyright (c) 2015 PlanGrid, Inc. All rights reserved.
```
Comment (josephdwyer) [inline PlanGrid.Api/RfiStatus.cs]: 2016

File: PlanGrid.Api/AttachmentUploadRequest.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+// </copyright>
+
+using Newtonsoft.Json;
+
+namespace PlanGrid.Api
+{
+    public class AttachmentUploadRequest
+    {
+        [JsonProperty("webhook_url")]
+        public string WebhookUrl { get; set; }
```
Comment (josephdwyer) [inline PlanGrid.Api/AttachmentUploadRequest.cs]: never used? what is this for?

Comment (josephdwyer) [inline PlanGrid.Api/AttachmentUploadRequest.cs]: never mind, I just realized this is the return value from the API Call

File: PlanGrid.Api.Tests/AttachmentTests.cs [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            IPlanGridApi client = PlanGridClient.Create();
+            AttachmentUploadRequest request = await client.CreateAttachmentUploadRequest(TestData.Project2Uid, new AttachmentUpload
+            {
+                ContentType = AttachmentUpload.Pdf,
+                Name = "test name",
+                Folder = "test folder"
+            });
+
+            Stream payload = typeof(AttachmentTests).Assembly.GetManifestResourceStream("PlanGrid.Api.Tests.TestData.Sample.pdf");
+            Attachment attachment = await client.Upload(request, payload);
```
Comment (josephdwyer) [inline PlanGrid.Api.Tests/AttachmentTests.cs]: While this is pretty easy for a client to understand, should we abstract the concept that there are two requests from end users?

e.g. `client.UploadAttachment(AttachmentUpload.Pdf, "test name", "test folder", payload);` which creates the request and puts the file.

File: PlanGrid.Api/MultipartUploader.cs [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+                foreach (IMultipartContent item in values)
+                {
+                    item.CreateContent(content);
+                }
+
+                HttpResponseMessage response = await client.PostAsync(new Uri(url), content, cancellationToken);
+                if ((int)response.StatusCode >= 400)
+                {
+                    string message = await response.Content.ReadAsStringAsync();
+                    throw new PhotoUploadException($"Error uploading attachment to S3: {message}");
```
Comment (josephdwyer) [inline PlanGrid.Api/MultipartUploader.cs]: not really a photo

**Comment by josephdwyer** [comment]
I assume you fixed `bbcf831` on master. Also, make sure to squash all of these guys.

**Pull request merged**